src/main/java/codechicken/lib/internal/network/CCLNetwork.java

```diff
@@ -15,15 +15,19 @@ public class CCLNetwork {
     //Client handled.
     public static final int C_ADD_LANDING_EFFECTS = 1;
     public static final int C_OPEN_CONTAINER = 10;
+    public static final int C_GUI_SYNC = 11;
+
+    //Server handled.
+    public static final int S_GUI_SYNC = 1;
```

Comment:
Generally i'd share the ID's between client and server if they are similar enough. Probably best to just pick `20` for both of these.